### ISSUE TITLE
Restructure swizzin file storing on FS.

### ISSUE THREAD

**Issue by flying-sausages** [swizzin/swizzin#526]
* Move `/install` to `/root/swizzin`
* move users to `/root/swizzin/users`
* change all functions to use these new locations

**Comment by liaralabs** [comment]
The panel needs to be able to read `/root/swizzin` if we move the install locks here. Not sure I feel good about giving the panel read access to `/root` :confused: We'll definitely need some discussion on the best way to handle this.

**Comment by flying-sausages** [comment]
very fair point

**Comment by flying-sausages** [comment]
Seems like we unanimously agree with using `/var/lib/swizzin/`, the `install/locks` can have `644` perms

**Comment by liaralabs** [comment]
Yeah, the locks are not sensitive data in the slightest, so that would work, as long as we secure the subdirs with appropriate permissions. That said, if we are encrypting user data here, this need becomes more of a "best practice" instead of "it's the only security we have to keep this data private"
